packages/react-dom/src/client/ReactDOMComponent.js

```diff
@@ -1239,3 +1138,5 @@ export function restoreControlledState(
       return;
   }
 }
+
+export {normalizeHTMLTextOrAttributeValue};
```

Comment:
This `normalizeHTMLTextOrAttributeValue` function is used in `ReactDOMHostConfig`, so exporting it here. It's not a `function` declaration because the function implementation gets substituted in `__DEV__`.